### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by brylie** [harishnavnit/NOC#1]
Under what license is this project released? If it is open-source, please provide a LICENSE file in the project root, so that people know their rights with the project sourcecode.

**Comment by tinvaan** [comment]
Yes, that is something that I have to look into. This is but a part of a larger project that we're working on. Didn't think this would garner any interest whatsover, and skipped adding the license. 

Thanks for reminding :-)
